indexinglanguage/src/main/javacc/IndexingParser.jj

```diff
@@ -33,6 +33,7 @@ import com.yahoo.text.StringUtilities;
 import com.yahoo.vespa.indexinglanguage.expressions.*;
```

Comment:
Unfortunately, atm we have two parsers. You need to also make these same additions in IndexingParser.ccc in schema-language-server.